content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -55,14 +55,17 @@ job:
     ...
     ```
 
-2.  `STUDIO_REPO_URL`: If you are running the experiment locally, or your
-    repository is on github.com, gitlab.com or bitbucket.org, you do not need to
-    set this environment variable. But if you are using some other Git provider,
-    then you should set the repository url in this format:
+2.  `STUDIO_REPO_URL`: If you are running the experiment locally, you do not
+    need to set this environment variable. But if you are running it in a CI
+    job, then you should set the repository url in this format:
     `{remote-type}:{namespace}/{repo-name}`. For example, for the
     `example-get-started` repository in the `iterative` namespace,
     `STUDIO_REPO_URL` should be set to the following value:
 
+    - If you are using GitHub.com, GitLab.com or Bitbucket.org, set it to
+      `github:iterative/example-get-started`,
```

Comment:
I'm not sure if these are the right formats. @daavoo @mvshmakov pls correct if needed.

Reply:
I think those formats would strictly work but only because of backward compatibility.

The currently expected format would be the return of `git ls-remote --get-url`, which would be something like:

`https://github.com/iterative/example-get-started.git`

Reply:
Thanks @daavoo. I've updated it.